web/models/registration.ex

```diff
@@ -0,0 +1,16 @@
+defmodule ElixirBlog.Registration do
+  import Ecto.Changeset, only: [put_change: 3]
+
+  alias ElixirBlog.Repo
+
+  def create(changeset) do
```

Comment:
That seems better. Thanks!

Btw, should a module like this be in web/models or should it be in lib? I wasn't entirely sure about that.